src/templates/filters.js

```diff
@@ -75,7 +75,7 @@ export const Filters = {
       emptyLinkMessage: i18n.translate("Get noticed: comment on posts you've read.", {
         comment: 'Notifications follows filter: no notifications',
       }),
-      emptyLink: 'https://wordpress.com/read/',
+      emptyLink: 'https://wordpress.com/activities/likes/',
```

Comment:
The PR description says this should be for the `likes` section, but it's in `follows` instead.

Though, both sections use the same empty message, and they should have the same link as well.

Reply:
Oh! Well spotted, and good suggestion; added the same to Likes as intended, and left this one here for Follows 👍